kornia/geometry/epipolar/projection.py

```diff
@@ -123,6 +123,9 @@ def KRt_from_projection(P: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor, to
     """
 
     assert P.shape[-2:] == (3, 4), "P must be of shape [B, 3, 4]"
+    assert len(P.shape) == 3
+
+    epsilon = torch.tensor([0.00001], device=P.device, dtype=P.dtype)
```

Comment:
I would suggest to define epsilon in the function signature as `float`. Something like this:

`def KRt_from_projection(P: torch.Tensor, eps: float = 1e-6) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor]:`